sdk/python/setup.py

```diff
@@ -20,6 +20,7 @@
 #VERSION = .... Change the version in kfp/__init__.py
 
 REQUIRES = [
+    'absl-py',
```

Comment:
Maybe we should limit the version as well?

Reply:
Good point. 

Just for reference, TFX currently sets: [`'absl-py>=0.9,<0.11'`](https://github.com/tensorflow/tfx/blob/77acc3ff231191597fead9a3f246a96294a883e1/tfx/dependencies.py#L55), while [0.11 was out in October 2020](https://pypi.org/project/absl-py/0.11.0/)

So how about we do `absl-py>=0.9,<=0.11`?

Reply:
SGTM.